src/shared/services/related-items-service.ts

```diff
@@ -28,6 +29,10 @@ export async function getRelatedItems(
 		const results: Avo.Search.Search = await response.json();
 		return results.results;
```

Comment:
Is onderstaand een optie? `results.results` klinkt namelijk wat awkward.

```
const { results }: Avo.Search.Search = await response.json();
return results;
```

Reply:
kzal het hernoemen naar:
```
const response: Avo.Search.Search = await response.json();
return response.results;
```